cylc/flow/task_trigger.py

```diff
@@ -147,7 +159,24 @@ def __str__(self):
         else:
             return '%s:%s' % (self.task_name, self.output)
 
-    __repr__ = __str__
+    def __repr__(self) -> str:
```

Comment:
Worth a quick test to keep the percentages up? I don't insist though.

Reply:
`repr` has little relevance to the user and is more important for debugging.

Reply:
Enough for me - I agree with you that it's a bit niff-naff, but I prefer to have an almost zero information test to a totally zero information coverage hole or exception.